src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown

```diff
@@ -1801,9 +1757,8 @@ Authorization: OAuth {access token}
 </Car>
 ```
 
-# <a name="objects"></a>Booking Object Elements
+## <a name="objects"></a>Booking Object Elements
 
-## Booking Object Elements - TMCs and Third-party Developers
 The booking elements contain many child elements. For ease of use, these elements are divided into the Core Elements, which are the most frequently used, and Additional Elements, which are not often used but are supported by the Itinerary web service. Some elements only appear if the travel supplier created the booking. Elements are marked as required if they must be supplied for a new booking.  
 
 **NOTE:** TripLink - Open Booking suppliers see a targeted subset of these fields. Refer to the documentation here for the TripLink - Open Booking supplier booking object elements.
```

Comment:
Looks like a link is missing here- "here" should point to this page: https://developer.concur.com/api-reference/travel/itinerary/trip/trip-resource.html